StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet.swift

```diff
@@ -236,6 +240,18 @@ public class CustomerSheet {
     var userCompletion: ((Result<PaymentOptionSelection?, Error>) -> Void)?
 }
 
+extension CustomerSheet {
+    static func allowsRemovalOfLastPaymentMethod(elementsSession: STPElementsSession, configuration: CustomerSheet.Configuration) -> Bool {
+        if !configuration.allowsRemovalOfLastSavedPaymentMethod {
+            // Merchant has set local configuration to false, so honor it.
+            return false
+        } else {
+            // Merchant is using client side default, so defer to CustomerSession's value
+            return elementsSession.paymentMethodRemoveLastForCustomerSheet
+        }
+    }
+}
```

Comment:
We pretty much have this function defined twice, why not just one that is
`static func allowsRemovalOfLastPaymentMethod(elementsSession: STPElementsSession, allowsRemovalOfLastSavedPaymentMethod: Bool) -> Bool`


Then `allowsRemovalOfLastPaymentMethod(elementsSession: elementsSession, allowsRemovalOfLastSavedPaymentMethod: configuration.allowsRemovalOfLastSavedPaymentMethod)`

Reply:
Hmm one accesses the customerSheet component, and one is used for PaymentSheet component.. but i agree, let me try to find a way to clean it up and make it more obvious.